multicluster/controllers/multicluster/member/node_controller.go

```diff
@@ -299,6 +307,11 @@ func (r *NodeReconciler) getGatawayNodeIP(node *corev1.Node) (string, string, er
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *NodeReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	serviceCIDR, err := common.DiscoverServiceCIDRByInvalidServiceCreation(context.TODO(), r.Client, r.namespace)
```

Comment:
I think you need to set the serviceCIDR as the one in MC config if it's no empty, otherwise, user's input will take no effect.

Reply:
Thanks for reminder, a check is added.